ui/src/app/pages/access/access-renewal.tsx

```diff
@@ -156,19 +169,28 @@ const CompletedButton = ({buttonText, wasBypassed, style}: CompletedButtonInterf
     <ClrIcon shape='check' style={{marginRight: '0.3rem'}}/>{wasBypassed ? 'Bypassed' : buttonText}
   </Button>;
 
+const wasBypassed = (modules, moduleName) => switchCase(moduleName,
+    [AccessModule.DATAUSERCODEOFCONDUCT, () => !!getAccessModuleBypassTime(modules, AccessModule.DATAUSERCODEOFCONDUCT)],
+    [AccessModule.COMPLIANCETRAINING, () => !!getAccessModuleBypassTime(modules, AccessModule.COMPLIANCETRAINING)],
+    // these cannot be bypassed
+    [AccessModule.PROFILECONFIRMATION, () => false],
+    [AccessModule.PUBLICATIONCONFIRMATION, () => false]);
+
+
 interface ActionButtonInterface {
-  isModuleExpiring: boolean;
-  wasBypassed: boolean;
+  modules: AccessModuleStatus[];
+  moduleName: AccessModule;
```

Comment:
would it make sense to change the interface to just provide a `module` instead of both the `modules` array and the `name` identifier?

In later usages, it looks like they're often paired together to pull out the correct module anyhow.

Reply:
Good idea.  I was able to update RenewalCard and a few other places to work this way.

Reply:
and now ActionButton too